src/main/kotlin/com/dashwave/plugin/PluginStartup.kt

```diff
@@ -53,11 +57,66 @@ class PluginStartup: StartupActivity {
         dwWindow.show()
         dwWindows[project.name] = dwWindow
         checkDW(project, dwWindow)
+
+//      start terminate gradle sync worker if pluginMode is 'workspace'
+        if(pluginMode == "workspace"){
+            terminateGradleSync(project.basePath, dwWindow)
+        }
+        terminateGradleSync(project?.basePath, dwWindow)
```

Comment:
why this line again?

Reply:
must have slipped into a rebase.
will remove